spec/parliament/request_spec.rb

```diff
@@ -63,9 +63,27 @@
   end
 
   describe '#get' do
```

Comment:
We have added tests to check that Parliament::Response responds to the delegated methods.